bundles/core/org.eclipse.smarthome.core.thing.xml/thing-description-1.0.0.xsd

```diff
@@ -38,7 +38,14 @@
         </xs:sequence>
         <xs:attribute name="id" type="config-description:idRestrictionPattern" use="required"/>
         <xs:attribute name="listed" type="xs:boolean" default="true" use="optional"/>
+        <xs:attribute name="extensible" type="thing-description:namespaceIdListRestrictionPattern" use="optional"/>
     </xs:complexType>
+    
+    <xs:simpleType name="namespaceIdListRestrictionPattern">
+        <xs:restriction base="xs:string">
+            <xs:pattern value="[A-Za-z0-9\-_.]+(,[A-Za-z0-9\-_.]+)*" />

```

Comment:
Also here (see above): Wouldn't be allowing an optional space be nice?

Reply:
yes, we might allow that, update is on the way.